dashboard/app/helpers/application_helper.rb

```diff
@@ -111,16 +111,15 @@ def external_oauth_sign_out_url(provider)
     end
   end
 
-  def meta_image_url(params)
-    level_source = params[:level_source]
-    if level_source
-      app = level_source.level.game.app
-    else
-      app = params[:app]
+  def meta_image_url
```

Comment:
Yeah in general I think method args are better than "magic" `@instance` vars in helpers.  See Will's refactor https://github.com/code-dot-org/code-dot-org/pull/1530 to remove magic from `LevelsHelper`.